lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1502,6 +1508,47 @@ def _node_generate_min_capability(
         min_cap: schema.NodeSpace = req_cap.generate_min_capability(azure_cap_copy)
         return min_cap
 
+    def _get_data_disk_list(
```

Comment:
The name can be `_generate_data_disks`. It means it's not "get", and it has logic to generate.

Reply:
update